generators/client/templates/react/src/main/webapp/app/config/notification-middleware.ts

```diff
@@ -13,12 +14,16 @@ export default () => next => action => {
    */
   return next(action).then(response => {
     if (action.meta && action.meta.successMessage) {
-      // toast(action.meta.successMessage, { type: toast.TYPE.SUCCESS });
+      if (typeof action.meta.successMessage === 'string') {
```

Comment:
I dont understand this, what is the else part for?

Reply:
it's a hack because in creation mode we pass in meta : 
`successMessage: translate('jhipsterApp.projet.created', { param: entity.id })`
but entity.id is undefined because the entity isn't created

Reply:
We should get the values from response headers then

Reply:
That means ee can move the translation step into the interceptor